kafka-connect-source/src/main/java/com/hazelcast/jet/tests/kafkaconnectsource/KafkaConnectSourceTest.java

```diff
@@ -0,0 +1,96 @@
+/*
+ * Copyright (c) 2008-2018, Hazelcast, Inc. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.hazelcast.jet.tests.kafkaconnectsource;
+
+import com.hazelcast.core.HazelcastInstance;
+import com.hazelcast.jet.Job;
+import com.hazelcast.jet.config.JobConfig;
+import com.hazelcast.jet.kafka.connect.KafkaConnectSources;
+import com.hazelcast.jet.pipeline.Pipeline;
+import com.hazelcast.jet.pipeline.Sink;
+import com.hazelcast.jet.pipeline.StreamSource;
+import com.hazelcast.jet.pipeline.StreamStage;
+import com.hazelcast.jet.pipeline.test.AssertionCompletedException;
+import com.hazelcast.jet.pipeline.test.AssertionSinks;
+import com.hazelcast.jet.tests.common.AbstractSoakTest;
+
+import java.net.MalformedURLException;
+import java.net.URL;
+import java.util.Properties;
+import java.util.concurrent.CompletionException;
+
+public class KafkaConnectSourceTest extends AbstractSoakTest {
+
+    private static final String CONNECTOR_URL = "https://repository.hazelcast.com/download"
```

Comment:
Can you please use our local repository?

Reply:
what do you mean local repository ?

Reply:
rev : 62052e37ab626ff865a3319e99368000b1956382